assets/blocks/index.js

```diff
@@ -3,7 +3,7 @@
  */
 import { __ } from '@wordpress/i18n';
 import { addFilter } from '@wordpress/hooks';
-import { registerBlockVariation } from '@wordpress/blocks';
+import { registerBlockVariation, registerBlockType } from '@wordpress/blocks';
```

Comment:
I think the `registerBlockType` is not needed here anymore, right?

Reply:
Not needed. You are right :) Good catch!